libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c

```diff
@@ -96,7 +96,7 @@ static TransportSocketStatus_t connectToServer( Socket_t tcpSocket,
 
 /*-----------------------------------------------------------*/
 
-int32_t SecureSocketsTransport_Send( NetworkContext_t * pNetworkContext,
+int32_t SecureSocketsTransport_Send( const NetworkContext_t * pNetworkContext,
```

Comment:
`transport_interface.h` was updated to remove the `const` fron send/recv. I know this is fixing MISRA 8.13, but this rule is just advisory. Other transport send/recv implementations (like in product 3) need to modify the `NetworkContext` object. I think this warning should just be suppressed.

Reply:
If that's our common consensus, then I will remove the ```const```.